monitor/prom/metrics.go

```diff
@@ -13,22 +13,24 @@ import (
 )
 
 const (
-	namespace      = "resticprofile"
-	backup         = "backup"
-	groupLabel     = "group"
-	profileLabel   = "profile"
-	goVersionLabel = "goversion"
-	versionLabel   = "version"
+	namespace          = "resticprofile"
+	backup             = "backup"
+	groupLabel         = "group"
+	profileLabel       = "profile"
+	goVersionLabel     = "goversion"
+	versionLabel       = "version"
+	resticVersionLabel = "resticversion"
```

Comment:
I don't think we need this constant anymore (sorry to be pedantic)

Reply:
You're right. Might be pedantic, but it is correct to do so.